### ISSUE TITLE
Konami Support?

### ISSUE THREAD

**Issue by samm329b** [marcrdavis/ArduinoTITO-PlayerTracking#5]
So far I have tried this out with WMS, Bally, and IGT having no issues, but the unit does not work with konami? Any chance you plan to add support for Konami?

**Comment by marcrdavis** [comment]
Hi samm329b - As I don't own a konami it would be hard to test and develop for. Most devices based on the SAS V6 protocol should work; I am guessing Konami is different. If there is any documentation on the protocol they are using I'd be happy to take a look.

**Comment by marcrdavis** [comment]
I have reports that is is working with Konami K2V, KP3 and Selixicon.

**Comment by samm329b** [comment]
Hello and thanks for the response. I have it plugged into a Konami k2v unit I’m testing with, I turned on the tilt mode for the communications when the credits were not adding, once I turned on the comm tilt about ever 3 seconds like clock work the machine would beep (like a door open beep) and the unit would said lost comm to the host. I have moved the unit over to a few other cabinets to confirm the tito is working fine, so I figured it had to be something within the delay commands in the code, I adjusted them and each time the same result, no beeping just a solid communication with host lost, so I reverted back to the original setting, and I’m back to where I started.
